vtr_flow/benchmarks/verilog/include/generic_definitions1.vh

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Seems like these two generic_definitions files should have comments at the top saying what they are used for.

I also think putting a short README.md file in the include/ directory saying what kinds of files should go in there would be good.